styles.css

```diff
@@ -416,3 +442,913 @@ textarea {
   background-color: #0d151e;
   border-radius: 10px;
 }
+
+/* MEDIUM SCREEN SIZES */
+
+@media screen and (min-width: 768px) and (max-width: 1080px) {
+  .myself {
+    background-image: none;
+    padding: 0;
+  }
+
+  .myselftext {
+    width: 40%;
+    margin: 0 5%;
+  }
+
+  .hrbottom {
+    margin-top: 10%;
+  }
+
+  .contactsection {
+    /* margin-bottom: 10%; */
+    padding: 4% 8%;
```

Comment:
- [ ] As suggested by the previous reviewer, kindly consider removing this commented line no 464 because it makes our codebase look unprofessional.